org.ekstep.mathtext-1.0/editor/mathtext.js

```diff
@@ -756,11 +767,15 @@ angular.module('org.ekstep.mathtext', [])
 
     $scope.getCursorPosition = function(e){
       var currentPosition = e.target.selectionEnd;
-      if(e.which == 8){
-        $scope.cursorPosition = currentPosition == 0 ? currentPosition : currentPosition -1;
+      if(!_.isUndefined($scope.selectionStart)){
+        $scope.cursorPosition = $scope.selectionStart ;
       }else{
-        $scope.cursorPosition = currentPosition == 0 ? currentPosition : currentPosition;
-      } 
+        if(e.which == 8){
+          $scope.cursorPosition = currentPosition == 0 ? currentPosition : currentPosition -1;
```

Comment:
Use just an IF condition here
if(currentPosition > 0)  currentPosition = currentPosition -1 ;

Reply:
[Pull request link](https://github.com/project-sunbird/sunbird-content-plugins/pull/374) : this PR has all the changes.